software/contrib/pong.py

```diff
@@ -0,0 +1,230 @@
+"""
+Pong
+author: Igor Nawrocki (github.com/Bugari)
+date: 2023-05-19
+labels: game
+
+EuroPi version of the classic Pong game.
+
+digital_in: unused
```

Comment:
[optional] Consider allowing and incoming trigger to set the pace for the game. This would allow the triggers to be tempo synced with whatever is going on in the rest of the patch.

Feel free to leave this off for future work.